bot/handlers/not_sending_videos/transcription_handler.py

```diff
@@ -20,23 +20,25 @@ class TranscriptionHandler(BotMessageHandler):
     def get_commands(self) -> List[str]:
         return ["transkrypcja", "transcription", "t"]
 
-    async def _do_handle(self, message: Message) -> None:
+    async def is_any_validation_failed(self, message: Message) -> bool:
         content = message.text.split()
         if len(content) < 2:
-            return await self._reply_invalid_args_count(message, get_no_quote_provided_message())
+            await self._reply_invalid_args_count(message, get_no_quote_provided_message())
+            return True
+        return False
 
-        quote = " ".join(content[1:])
+    async def _do_handle(self, message: Message) -> None:
+        quote = " ".join(message.text.split()[1:])
         result = await TranscriptionFinder.find_segment_with_context(quote, self._logger, context_size=15)
 
         if not result:
             return await self.__reply_no_segments_found(message, quote)
 
-        context_segments = result["context"]
-        response = get_transcription_response(quote, context_segments)
+        response = get_transcription_response(quote, result)
         await self.__reply_transcription_response(message, response, quote)
 
     async def __reply_no_segments_found(self, message: Message, quote: str) -> None:
-        await message.answer(get_no_segments_found_message(quote))
+        await message.answer(get_no_segments_found_message(quote), parse_mode="Markdown")
```

Comment:
jestem prawie pewny ze o tym gadalismy xD anwser markdown czy cos mialo byc

Reply:
a dobra jestem taki sprytny że zmieniłem to w tej jednej klasie co zaznaczyłeś ostatnio i musiałem nie wpaść na to żeby zmienić to wszędzie XD